stdlib/public/SwiftShims/RefCount.h

```diff
@@ -432,7 +431,14 @@ class RefCountBitsT {
   }
 
   LLVM_ATTRIBUTE_ALWAYS_INLINE
-  RefCountBitsT(RefCountBitsT<RefCountIsInline> newbits) {
+  RefCountBitsT(int dummy, RefCountBitsT<RefCountIsInline> newbits) {
```

Comment:
Does this work if you do something more clever like:

    RefCOuntBitsT(RefCountBitsT<RefCountIsInline> newBits, int dummy = 0);